packages/deployer/internal/contract-helper.js

```diff
@@ -40,22 +41,16 @@ async function getModulesSelectors() {
 }
 
 function findDuplicateSelectors(selectors) {
-  const duplicates = selectors
-    .map((s) => s.selector)
-    .filter((s, index, selectors) => selectors.indexOf(s) !== index);
```

Comment:
We were repeating this pattern for getting duplicate values on an array, which IMHO is somewhat hard to read and understand. So, I moved it to a helper filter function on `core-js`.

Also, this had a bug were if the  repetition was present more than two times, it would repeat the error. For example, for  `[1,1]` would return `[1]`, but  for `[1,1,1] => [1,1]` and so on. (The fix  can be seen on the new `array-filters.js` file)